MarketplaceService/Exceptions/ProductNotFoundException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace MarketplaceService.Exceptions
+{
+    [Serializable]
+    public class ProductNotFoundException: Exception
+    {
+        public ProductNotFoundException()
+            : base("All fields have to be filled out.")
```

Comment:
Ik neem aan dat dit vergeten was met de copy paste om aan te passen naar product was not found of iets dergelijks :)

Reply:
Oh derp, ja dat klopt.